packages/rx_bloc_cli/mason_templates/bricks/lib_pin_code/__brick__/lib/lib_pin_code/views/verify_pin_code_page.dart

```diff
@@ -0,0 +1,129 @@
+{{> licence.dart }}
+
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:widget_toolkit/models.dart';
+import 'package:widget_toolkit/ui_components.dart';
+import 'package:widget_toolkit_biometrics/widget_toolkit_biometrics.dart';
+import 'package:widget_toolkit_pin/widget_toolkit_pin.dart';
+
+import '../../base/common_blocs/coordinator_bloc.dart';
+import '../../l10n/l10n.dart';
+import '../bloc/update_and_verify_pin_bloc.dart';
+import '../models/pin_code_arguments.dart';
+import '../services/update_and_verify_pin_code_service.dart';
+
+class VerifyPinCodePage extends StatefulWidget {
```

Comment:
Same here - code is quite similar to the `create_pin_page` and `update_pin_page`.

Reply:
I have previously tried that, but was not successful.